kitsune/search/v2/documents.py

```diff
@@ -15,30 +19,41 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
-    updated = field.Date()
+    """ES Document for Knowledge Base.
+
+    Each document in ES is built on top of a parent KB article.
+    All the children are included in the same document.
+    """
 
+    updated = field.Date()
     product_ids = field.Keyword(multi=True)
     topic_ids = field.Keyword(multi=True)
+    locale = field.Keyword()
+    category = field.Keyword()
+    recent_helpful_votes = field.Integer()
```

Comment:
I'm not sure it's all that useful having recent helpful votes of just the parent.

Reply:
The idea (for now) it's to retrieve the helpful_votes of the children if we want to aggregate on them. If this is still the case we need to store the helpful votes of the parent document too. Otherwise we will add all of them as we did for the other fields